app/views/samples/_filters.haml

```diff
@@ -25,3 +25,20 @@
             = cdx_select name: "specimen_role", value: params["specimen_role"] do |specimen_role|
               - specimen_role.item "", "All specimen roles"
               - specimen_role.items Sample.specimen_roles.map{|item| {id: item[:id], description: item[:description].truncate(30) }}, :id, :description
+          .filter-group
+            .filter
+              %label.block Modified From
+              %input.input-large{type: "date", name: "updated_at_from", value: params["updated_at_from"]}
+            .filter
+              %label.block Modified To
+              %input.input-large{type: "date", name: "updated_at_to", value: params["updated_at_to"]}
+            .filter
+              %label.block Modified
+              = cdx_select name: "modified", value: params["modified"] do |modified|
+                - modified.item "", "All"
+                - modified.item Date.today, "Today"
+                - modified.item Date.today - 1.days, "Yesterday"
+                - modified.item Date.today - 7.days, "7 Days"
+                - modified.item Date.today - 30.days, "30 Days"
+                - modified.item Date.today - 60.days, "60 Days"
+                - modified.item Date.today - 365.days, "Last Year"
```

Comment:
We can be even more cool in Rails, for example: `1.day.ago`, `1.week.ago`, `1.month.ago`, `1.year.ago`

Reply:
good to know....but the days of this dropdown are counted...I removed it :)